### DISCUSSION TITLE
Display actual dates/times instead of approximated durations.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#12553] [Projects and Issues]
All dates/times are displayed are in terms of approximated durations such as "4 days ago".  To see the exact/date/time, You have to hover over that field then after a brief delay the actual date/time pops up until you move away from it. This seems like poor usability design especially when trying to figure out actual timelines from a screen that contains multiple entries, so having to hover over, wait, then remember or write down each actual date/time. Also at least in my case, there is never a benefit to seeing an approximated time over an accurate one.

I would like to propose that GitHub either change the website so that exact dates/times (i.e. what you see when you hover) are displayed by default, or perhaps add an option that allows individual users to select which style they prefer to see by default.

**Comment by dprentis** [comment]
Couldn't agree more. I'd rather directly see when something happened, rather than how long ago it happened.

**Comment by nekoniaow** [comment]
Absolutely, when looking for a commit by date this increases the time and effort needed by at least one order of magnitude (and I am being kind).

This requires to stop at a given commit, hover over its "years ago" item which takes several seconds instead of the instantaneous glance a display of the date would offer.

Usability wise, this is quite horrendous. Moreover, it boggles the mind that "time ago" was deemed a useful information at all.
Human beings are capable at a glance to realize whether any date is a few days/months/years in the past without having to resort to a calculator. Why would anyone think they needed assistance in this matter is really the weird part here.

A setting in one's profile to change this seems to be an appropriate addition.

**Comment by drewcdisc** [comment]
Ditto. Bump. 

Please fix this! I have non-technical users who are wondering about this. Usability losses like this diminish reputation for GitHub. I need my non-techies to feel comfortable in there, and even non-techies need exact dates.

**Comment by junderhill** [comment]
100% or at least give the option to set your preference within settings.
Relative timestamps are awful

**Comment by iamron** [comment]
One more vote. I don't want a dumbed down version that hides critical information like this.

When we're look at the commit history or whatever, generally we're after the precise date time. Hover over kind of works but I only found out about it from this discussion thread.

It's not intuitive and highly frustrating.

**Comment by ennuite** [comment]
I can't believe this hasn't been addressed yet.
